babel.config.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Legal approved of this change I assume? Sometimes changing copyright has legal ramifications (but I doubt it matters here).

Reply:
I'm using AMP HTML authors for everything AMP related.